examples/tutorials/advanced/grid_equalization.py

```diff
@@ -20,7 +20,7 @@
 grid = pygmt.datasets.load_earth_relief(
     resolution="03s", region=[-119.825, -119.4, 37.6, 37.825]
 )
-grid_dist = pygmt.grd2xyz(grid=grid, output_type="pandas")["elevation"]
+grid_dist = pygmt.grd2xyz(grid=grid, output_type="pandas")["z"]
```

Comment:
Also need to update https://github.com/GenericMappingTools/pygmt/blob/8b2a74ca8e68c1d7e34e46b089bf522e7ba683f3/pygmt/src/grd2xyz.py#L142.